src/main/java/io/jenkins/plugins/folderauth/acls/AbstractAcl.java

```diff
@@ -15,6 +15,40 @@
 import java.util.concurrent.ConcurrentHashMap;
 
 abstract class AbstractAcl extends SidACL {
+
+    private static ConcurrentHashMap<Permission, Set<Permission>> implyingPermissionsCache = new ConcurrentHashMap<>();
```

Comment:
Will this map be persisted?

Reply:
No it shouldn't be persisted. It's `static`.